app/views/declaration/draft_confirmation_page.scala.html

```diff
@@ -28,17 +30,26 @@
         <h1 class="heading-xlarge">
             @messages("confirmation.draft.title")
         </h1>
-        <p class="font-large">@messages("confirmation.draft.info", flash.get(FlashKeys.expiryDate).getOrElse("-"))</p>
+        <p class="font-large">
+         @messages("confirmation.draft.info")
+        </p>
+        <p id="draft_confirmation-expiry" class="font-large">
+        @{
+            flash.get(FlashKeys.expiryDate)
```

Comment:
are you storing it in the `flash` store?

Reply:
Yes, the Navigator handles that aspect